website_seo_redirection/models/website_seo_redirection.py

```diff
@@ -127,3 +159,32 @@ def redirect_auto(self, path=None, code=301, website=None, rerouting=None):
             dict(request.httprequest.args),
             True,
```

Comment:
@lasley, @Yajo Hi, I just tested the add-on for odoo 9 and it works so far, really good and simple solution. But all the redirections have only the status code 200 instead of 301.
In the function redirect_auto() you call local_redirect() at the end with only 4 parameters which results in keep_hash=True. Is it supposed to work in this way?
For testing I used keep_hash=False which results in a status code 301 for the redirections.